packages/fuels-rs/Cargo.toml

```diff
@@ -13,3 +13,5 @@ fuels-abigen-macro = { version = "0.9.0", path = "../fuels-abigen-macro" }
 fuels-contract = { version = "0.9.0", path = "../fuels-contract" }
 fuels-core = { version = "0.9.0", path = "../fuels-core" }
 fuels-signers = { version = "0.9.0", path = "../fuels-signers" }
+# This probably shouldn't be here, SDK users should put it in their dev-dependencies.
+fuels-test-helpers = { version = "0.9.0", path = "../fuels-test-helpers" }
```

Comment:
This version `0.9.0` doesn't actually exist on crates.io, and never will if the first version of this crate is `0.9.1`.

Reply:
fixed in [5c10cca](https://github.com/FuelLabs/fuels-rs/pull/228/commits/5c10ccaadd0423aa3da336faa20eac4dd088689d), also closes #221